MSDOS/Makefile-MSDOS

```diff
@@ -0,0 +1,71 @@
+# $Header: /home/henk/CVS/dealer/MSDOS/Makefile-MSDOS,v 1.2 1999/07/03 19:59:56 henk Exp $
+
+CC      = gcc
+CFLAGS = -Wall -pedantic -O2 -I. -DNDEBUG -DMSDOS -c
+FLEX    = flex
+YACC    = yacc
+
+PROGRAM  = dealer
+TARFILE  = ${PROGRAM}.tar
+GZIPFILE = ${PROGRAM}.tar.gz
+
+SRC  = dealer.c pbn.c  c4.c getopt.c pointcount.c
+LSRC = scan.l
+YSRC = defs.y
+HDR  = dealer.h tree.h
+
+OBJ  = dealer.o defs.o pbn.o c4.o getopt.o pointcount.o
+LOBJ = scan.c
+YOBJ = defs.c
+
+
+dealer: ${OBJ} ${LOBJ} ${YOBJ}
+	${MAKE} -C Random lib
+	$(CC) -o $@ ${OBJ} -L./Random -lgnurand
+
+clean:
+	rm -f ${OBJ} 
+	${MAKE} -C Examples clean
+	${MAKE} -C Random   clean
+
+tarclean: clean
+	rm -f dealer
```

Comment:
Same questions here about why `make clean` doesn't delete `dealer` but `make tarclean` does, and how `make tarfile` depends on `make tarclean` which might delete the program trying to be tarred up. I dunno, I'm not gonna touch the MSDOS stuff.